SCClassLibrary/Common/Audio/SynthDef.sc

```diff
@@ -586,13 +587,27 @@ SynthDef {
 
 	doSend { |server, completionMsg|
 		var bytes = this.asBytes;
+		var path;
+		var resp, syncID;
+
 		if (bytes.size < (65535 div: 4)) {
-			server.sendMsg("/d_recv", bytes, completionMsg)
-		} {
+ 			server.sendMsg("/d_recv", bytes, completionMsg)
+ 		} {
```

Comment:
can you please undo these whitespace changes?

Reply:
done, thanks!